contributing.md

```diff
@@ -300,11 +300,21 @@ Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_mediu
 
 Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init), [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/), or [pnpm](https://pnpm.io) to bootstrap the example:
 
+### NPM
```

Comment:
Let's update this document as well.

You can drop the new lines between the ending \``` and a ```bash starting code fence to spare 2 extra lines as well.

I.e. for example:

![image](https://user-images.githubusercontent.com/18369201/178261183-91b9b2bc-c776-4f23-a9cc-16fe13735d25.png)

Reply:
@balazsorban44 any chance of a new review?